README.md

```diff
@@ -57,6 +57,7 @@ List of Base Layers
 | apache-php | [Repo](https://github.com/juju-solutions/layer-apache-php.git) | [Docs](https://github.com/juju-solutions/layer-apache-php.git#readme) | Apache PHP base layer | Configurable base for Apache PHP charms |
 | apache-wsgi | [Repo](https://git.launchpad.net/~jacekn/charms/+source/apache-wsgi) | [Docs](https://git.launchpad.net/~jacekn/charms/+source/tree/README.md) | Apache WSGI base layer | Configurable base for Apache WSGI charms |
 | apt | [Repo](https://git.launchpad.net/layer-apt) | [Docs](https://github.com/stub42/layer-apt#readme) | Apt layer | Easily deal with apt sources and deb packages |
+| aws-iam | [Repo](https://github.com/juju-solutions/charm-aws-iam) | [Docs](https://github.com/juju-solutions/charm-aws-iam#readme) | AWS-IAM | AWS-IAM subrodinate charm for Kubernetes charm integration. |
```

Comment:
Why is the charm being added here?  Is it intended to be used as a base layer in addition to being a subordinate on its own?

Reply:
:man_facepalming: I'll pull that.